### PR TITLE
Sort dispute outcome order

### PR DESCRIPTION
**Pull request by bthaile**
only Yes No and categorical markets sort correctly

https://github.com/augurproject/augur/issues/3628

I'll get another PR in for scalar markets. 

still waiting for sdk changes for how payout numerators are converted to outcomes. 

have a few fixes on this PR

### CODE REVIEW

**Review by phoebemirman — CHANGES_REQUESTED** [review]
Screen Shot 2019-09-16 at 2 18 09 PM
Screen Shot 2019-09-16 at 2 17 54 PM
shouldn't tentative winner be showing up first?

**Review by ghost — COMMENTED** [review]
This pull request has been automatically deployed to [FeaturePeek](https://featurepeek.com). 🆒

 This link will always point to this PR's latest changes: https://peek.run/93ph301

File: packages/augur-ui/src/modules/markets/selectors/market.ts [hunk 1]
```diff
@@ -52,3 +59,57 @@ export const selectSortedMarketOutcomes = (marketType, outcomes) => {
...
+    if (sorted.length === 0) return filteredSortedOutcomes;
+    const result = sorted.reduce(
+      (p, s) => [...p, { ...genericScalarOutcome, id: s.outcome }],
+      []
+    );
+    return result;
+  }
+
+  if (stakes.length > 0) {
+    console.log("yes no, cat")
```
Comment (JohnDanz) [inline packages/augur-ui/src/modules/markets/selectors/market.ts]: remove this log

File: packages/augur-ui/src/modules/markets/selectors/market.ts [hunk 2]
```diff
@@ -52,3 +59,57 @@ export const selectSortedMarketOutcomes = (marketType, outcomes) => {
...
+    stakes.map(s => console.log(JSON.stringify(s)));
```
Comment (JohnDanz) [inline packages/augur-ui/src/modules/markets/selectors/market.ts]: and this one

**Review by JohnDanz — APPROVED** [review]
just remove those console logs and looks good

**Comment by bthaile** [comment]
need alex's PR merged to use the new properties on DisputeInfo. closing

**Review by phoebemirman — APPROVED** [review]

**Pull request merged**